src/server.rs

```diff
@@ -306,6 +297,27 @@ pub fn configure_routes(cfg: &mut web::ServiceConfig) {
         .route("", web::head().to(HttpResponse::MethodNotAllowed));
 }
 
+/// Retrieves all configured tokens.
+#[allow(deprecated)]
+fn get_tokens(config: &Config) -> Option<Vec<String>> {
+    if config.server.auth_token.is_some() || config.server.auth_tokens.is_some() {
+        let mut merged: Vec<String> = vec![];
+        if let Some(tokens) = &config.server.auth_tokens {
+            merged = tokens.clone();
+        }
+        if let Some(token) = &config.server.auth_token {
+            log::warn!("[server].auth_token is deprecated, please use [server].auth_tokens");
+            merged.insert(0, token.to_string());
+        }
+        if let Ok(env_token) = env::var(AUTH_TOKEN_ENV) {
+            merged.insert(0, env_token);
+        }
+        Some(merged)
+    } else {
+        None
+    }
+}
```

Comment:
Please move this method to make it a part of `Config`. I think calling `config.get_tokens()` is cleaner.

Reply:
I'll look into it.

Reply:
Done. Btw, I have done the same for the deprecation warning function.